### ISSUE TITLE
Page not found (404)

### ISSUE THREAD

**Issue by JorgeSevilla** [geex-arts/django-jet#289]
Hi, I make download of the project later, I try to run the project and I have the next issue when try to surf of the dashboard.
Using the URLconf defined in jet.tests.urls, Django tried these URL patterns, in this order:
url: http://127.0.0.1:8000/admin/jet
    ^jet/
    ^jet/dashboard/
    ^admin/doc/
    ^admin/ ^$ [name='index']
    ^admin/ ^login/$ [name='login']
    ^admin/ ^logout/$ [name='logout']
    ^admin/ ^password_change/$ [name='password_change']
    ^admin/ ^password_change/done/$ [name='password_change_done']
    ^admin/ ^jsi18n/$ [name='jsi18n']
    ^admin/ ^r/(?P<content_type_id>\d+)/(?P<object_id>.+)/$ [name='view_on_site']
    ^admin/ ^auth/user/
    ^admin/ ^tests/testmodel/
    ^admin/ ^auth/group/
    ^admin/ ^sites/site/
    ^admin/ ^tests/relatedtotestmodel/
    ^admin/ ^(?P<app_label>auth|tests|sites)/$ [name='app_list']

The current path, admin/jet, didn't match any of these.
Somebody can help me please.

**Comment by zodman** [comment]
follow the install instructions

**Comment by JorgeSevilla** [comment]
Thanks for the answer.



2018-01-31 16:05 GMT-02:00 Andres Vargas <notifications@github.com>:

**Comment by zodman** [comment]
@f1nality  close the issue

**Comment by dillonreyna** [comment]
Django 2.0

Followed the install instructions - issue persists.

```
INSTALLED_APPS = [
    'jet.dashboard',
    'jet',
    'django.contrib.admin',
```

```
TEMPLATES = [
    {
        'BACKEND': 'django.template.backends.django.DjangoTemplates',
        'DIRS': [],
        'APP_DIRS': True,
        'OPTIONS': {
            'context_processors': [
                'django.template.context_processors.debug',
                'django.template.context_processors.request',
                'django.contrib.auth.context_processors.auth',
                'django.contrib.messages.context_processors.messages',
            ],
        },
    },
]
```

```
urlpatterns = [
    path('jet/', include('jet.urls', 'jet')),
    path('jet/dashboard/', include('jet.dashboard.urls', 'jet-dashboard')),
    path('admin/', admin.site.urls),
```

**Comment by SalahAdDin** [comment]
I had no problems with `django` 2.0.

**Comment by adriadrop** [comment]
Have the same problem :(  django 2.1

**Comment by CodeSpent** [comment]
Third issue I've seen opened on this where most say "follow the instructions". Many of us have and are still faced with this issue. 

Here is the debugger's output:
`

Using the URLconf defined in dsg-it-database.urls, Django tried these URL patterns, in this order:

    ^$ [name='index']
    ^inventory/$ [name='inventory']
    ^access_denied/$ [name='access_denied']
    ^inventory/add_item/$ [name='add_item']
    ^inventory/add_onhand/(?P<query>\w+)/$ [name='add_onhand']
    ^inventory/add_onhand$ [name='add_onhand']
    ^inventory/view_item/(?P<query>\w+)/$ [name='view_item']
    accounts/
    ^stores/$ [name='stores']
    ^stores/edit_life_safety/(?P<query>\w+)/$ [name='edit_life_safety']
    ^stores/(?P<query>\w+)/$ [name='view_store']
    ^stores/search/(?P<query>\w+)/$ [name='store_search']
    ^equipment/servers/$ [name='view_all_servers']
    admin/
    ^jet/ ^add_bookmark/$ [name='add_bookmark']
    ^jet/ ^remove_bookmark/$ [name='remove_bookmark']
    ^jet/ ^toggle_application_pin/$ [name='toggle_application_pin']
    ^jet/ ^model_lookup/$ [name='model_lookup']
    ^jet/ ^jsi18n/$ [name='jsi18n']
    ^jet/dashboard/

The current path, jet/dashboard, didn't match any of these.
`

Here is my **urls.py**:
`from django.contrib import admin
from django.urls import path, include, re_path
from depot import views
   
urlpatterns = [
    path('', include('depot.urls')),
    path('', include('stores.urls')),
    path('admin/', admin.site.urls),
    re_path(r'^jet/', include(('jet.urls', 'jet'))),
    re_path(r'^jet/dashboard/', include(('jet.dashboard.urls', 'jet-dashboard'))),
    
    ]`

**Comment by SalahAdDin** [comment]
You must put them before to the admin urls if i'm not wrong.

**Comment by CodeSpent** [comment]
> 
> 
> You must put them before to the admin urls if i'm not wrong.

You are correct, this was my issue. I've helped 2 others who missed this note as well. So future readers, verify your jet urls are before admin urls.
